Source/WebCore/Modules/url-pattern/URLPatternParser.cpp

```diff
@@ -314,7 +315,7 @@ static String escapeRegexStringForCharacters(std::span<const CharacterType> char
     result.reserveCapacity(characters.size());
 
     for (auto character : characters) {
-        if (std::find(regexEscapeCharacters.begin(), regexEscapeCharacters.end(), character) != regexEscapeCharacters.end())
+        if (std::ranges::find(regexEscapeCharacters, character) != regexEscapeCharacters.end())
```

Comment:
Not sure where we are on c++23, but `std::ranges::contains` would work here.

Reply:
Unfortunately std::ranges::contains is C+23, not C++20 and is not supported yet by all configurations. I have tried it recently and it built locally but not on EWS.